vmm/src/config.rs

```diff
@@ -534,6 +534,24 @@ impl MemoryConfig {
             zones,
         })
     }
+
+    pub fn get_size(&self) -> u64 {
```

Comment:
Please call this `total_ram_size()` instead of `get_size()`.

Reply:
Changed.

Reply:
This is a memory struct.  So I changed this function name to "total_size".